packages/compass-sidebar/package.json

```
@@ -70,7 +70,7 @@
     "redux-thunk": "^2.4.2"
   },
   "devDependencies": {
-    "mongodb-data-service": "^22.19.0",
+    "mongodb-data-service": "^22.18.1",
```

```suggestion
    "mongodb-data-service": "^22.19.0",
```
We can probably keep the most recent one?

---

Yeah, I don't get it ... why did this script even modify these in the first place 🤷‍♀️

---

Yea this happened to me as well when I tried updating electron once with this script. I believe the node-abi package update changes were also automatic, right?